### ISSUE TITLE
data_analysis

### ISSUE THREAD

**Issue by LDJ20210926** [skwfz/critical_look_causal_dlvms#1]
Can I ask a question about why we never deal with continous features,just like normalization?

**Comment by severi-rissanen** [comment]
Hi, can you elaborate what you mean by continuous features? The X and T and Y variables? If so, we did do experiments where those are continuous with the "linear-Gaussian" data.

**Comment by LDJ20210926** [comment]
thank you very much! And another question: Did you run jobs datasets , if did, can your share your idea

**Comment by severi-rissanen** [comment]
I didn't do experiments on the jobs data, unfortunately. The ones detailed in the paper and in this repo are all. But I would imagine that jobs wouldn't be very different, I'm sure you can find details on handling that data from some other papers that use it.

**Comment by LDJ20210926** [comment]
unfortunately，I never found details about the standard of jobs, anyway, another quetion that I always think is that the CEVAE model whether breaks the rule of unconfoundness which is from the three rules about casual inference, can you explian?

**Comment by severi-rissanen** [comment]
Yeah I can try, I assume that by the 'rule of unconfoundness' you mean that there are no unobserved confounders that we don't account for. In contrast, the model does assume a causal structure and an unobserved confounder, and tries to account for that. This is possible to do, e.g., in the linear-Gaussian case where we know that the unobserved confounder has a very specific structure and we have proxy variables (see "effect restoration"). One of the motivations for CEVAE was that maybe it can relax these assumptions further and take care of a wider range of unobserved confounders with proxy variables. In our paper, we show that this doesn't work in general.

**Comment by LDJ20210926** [comment]
firstly,your answer is very reasonable; I can understand that just in the special case: the linear-Gaussian case , we can relax the unobserved confounder, but in other cases, it just doen't work. In other words, CEVAE breaks the rule of unconfoundness just in the situation of datasets that obey the linear-Gaussian distribution, can I understand correctly ?

**Comment by severi-rissanen** [comment]
Yes, it mostly works in the linear-Gaussian where the unobserved confounder is 1D and Gaussian distributed. In other cases, it often doesn't work correctly. 

But I'm not entirely sure what you mean by the rule of unconfoundness: The goal of causal inference is usually to account for confounders and if they don't exist, there's not much to do. I think that the question is not whether there is a confounder, but instead whether we know about it and whether we are able to account for it. The finding in our paper is that CEVAE is not able to account for them in many cases.

**Comment by LDJ20210926** [comment]
oh,yeah,I am so sorry not explaining what's the meaning of unconfoundedness. In my mind, the unconfoundedness is the same as ignorability from Pearl's idea, and more detailly, it means Given  covariates X,  Y and T are independent

**Comment by LDJ20210926** [comment]
And Another quetion is that in my mind, Heterogeneity and selection bias are the same thing?

**Comment by severi-rissanen** [comment]
Alright good, that's the idea that I assumed. Not sure what is heterogeneity in your context; I haven't been working with causality theory for a while now. Maybe you can look up the definitions?
